extensions/kill-process/src/index.tsx

```diff
@@ -88,23 +141,25 @@ export default function ProcessList() {
     >
       {state
         .filter((process) => {
-          if (query == null) {
+          if (query == "" || query == undefined) {
```

Comment:
I think `== null` checks for `undefined` as well, but `== undefined` doesn't check for `null` too.

Reply:
Changed to `if (!query)`, is it acceptable?

Reply:
Sounds reasonable, I'll fix it soon!